### DISCUSSION TITLE
Simple schedule with 24 hour interval become 1 millisecond interval

### DISCUSSION THREAD

**Discussion by ytcitsupport-jlin** [quartznet/quartznet#1668] [Q&A]
**Describe the bug**

A simple schedule becomes 1 millisecond when set interval with 24 hours.

**Version used**

Quartz.AspNetCore 3.4.0

**To Reproduce**

```
ITrigger trigger = TriggerBuilder.Create()
                    .WithIdentity("Sample", "DEFAULT")
                    .StartNow()
                    .WithSimpleSchedule(x => x.WithMisfireHandlingInstructionIgnoreMisfires()
                        .WithIntervalInHours(24)
                        .RepeatForever())
                    .ForJob(job)
                    .Build();
```

**Expected behavior**

Expect the trigger to be fired every 24 hours, but the scheduler saves the trigger with a 1-millisecond interval.

**Additional context**
DB: SQL Server

**Comment by jafin** [comment]
@ytcitsupport-jlin  If I understand, because you have specified `StartNow()` (which I think is also the default start) the trigger will fire `now` and then again every 24 hours from now.
If you wanted the trigger to fire in 24hours, perhaps 

```c#
.StartAt(DateTime.UtcNow.AddHours(24).AddSeconds(-1))
```

Here is what I see with a trigger configured similar to what you post after waiting 5 seconds.
Trigger has fired. next fire time is 24hrs from now.
![image](https://user-images.githubusercontent.com/127927/169682358-8633368a-acfc-4b66-9f76-d9f9e68af22f.png)

And here is what i see if I trigger specifying a StartAt
```c#
.StartAt(DateTime.UtcNow.AddHours(24).AddSeconds(-1))
```
![image](https://user-images.githubusercontent.com/127927/169682379-b5af63d6-a112-49e1-aeee-cbf7921454eb.png)

Trigger has not started, and is scheduled 24 hours from now.
